### PR TITLE
Run unit tests

### PR DESCRIPTION
**Pull request by ggravlingen**
I'm still new to Github action but it doesn't look like we're actually running the test suite?

### CODE REVIEW

**Comment by ggravlingen** [comment]
Looks like not all tests are run but tests are being returned as passed?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -29,3 +29,6 @@ jobs:
       - name: Check formatting with black
         run: |
           black pytradfri examples tests --check
+      - name: Run unit tests
+        run: |
+          python -m unittest discover tests
```
Comment (MartinHjelmare) [inline .github/workflows/test.yml:34]: We should use pytest as test runner.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -29,3 +29,6 @@ jobs:
       - name: Check formatting with black
         run: |
           black pytradfri examples tests --check
+      - name: Run unit tests
+        run: |
+          pytest tests
```
Comment (MartinHjelmare) [inline .github/workflows/test.yml:34]: We need to install the `pytradfri` local package before we run the tests.

Comment (ggravlingen) [reply .github/workflows/test.yml:35]: Are you sure about this? Don't think we did this when running unit tests in the previous CI/CD setup.

**Comment by ggravlingen** [comment]
I accidentally commited one of the changes in the test files directly to master rather than to this branch (I'm doing this from the browser). Sorry about that. The same commit has also been done to this PR.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -23,6 +23,7 @@ jobs:
         run: |
           pip install -r requirements.txt
           pip install -r requirements_test.txt
+          python setup.py install
```
Comment (MartinHjelmare) [inline .github/workflows/test.yml:26]: `pip install .` also works.

File: .github/workflows/test.yml (hunk 2 above)
Comment (MartinHjelmare) [reply .github/workflows/test.yml:35]: The best is to install the package in a virtual environment and run the tests there. `tox` is a tool that does this.

https://docs.pytest.org/en/stable/goodpractices.html

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**